include/my_robot.h

```diff
@@ -0,0 +1,74 @@
+/** \file my_robot.h
```

Comment:
I'm afraid that this will not work for [Eclipse plugin](https://github.com/c4ev3/EV3-Eclipse-Plugin) users (it works for the Makefile project though). I'm not 100% sure, but it seems that the plugin ships precompiled `libev3api.a` and it does not recompile EV3-API from the source code. This would mean that the values in `ev3_robot.c` would always get initialized to the default values.